src/main/java/com/jfrog/ide/idea/ui/configuration/JFrogGlobalConfiguration.form

```diff
@@ -68,32 +68,32 @@
       </component>
       <vspacer id="aece4">
         <constraints>
-          <grid row="14" column="1" row-span="3" col-span="2" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false"/>
+          <grid row="15" column="1" row-span="3" col-span="2" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false"/>
         </constraints>
       </vspacer>
       <vspacer id="75e1b">
         <constraints>
-          <grid row="16" column="0" row-span="3" col-span="3" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false">
+          <grid row="17" column="0" row-span="3" col-span="3" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false">
             <preferred-size width="133" height="14"/>
           </grid>
         </constraints>
       </vspacer>
       <component id="10519" class="javax.swing.JLabel">
         <constraints>
-          <grid row="14" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+          <grid row="15" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
         </constraints>
         <properties>
           <enabled value="false"/>
-          <text value="A glob pattern used to exclude specific paths from being scanned by JFrog Xray."/>
+          <text value="For example, package.json files under directories named testdata will not be scanned."/>
         </properties>
       </component>
       <component id="e7f46" class="javax.swing.JLabel">
         <constraints>
-          <grid row="15" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+          <grid row="16" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
         </constraints>
         <properties>
           <enabled value="false"/>
-          <text value="For example, package.json files under directories named testdata will not be scanned."/>
+          <text value="The excluded paths supported only by npm and Go projects."/>
```

Comment:
The excluded paths supported only by npm and Go projects.
-->
This is supported only by npm and Go projects.